core/templates/dev/head/css/oppia.css

```diff
@@ -358,6 +358,21 @@ p:last-child {
   margin: 1.4em 0;
 }
 
+/* This will prevent the enlargment of the text in the RTE. A span tag gets
+   introduced when a user selects the text from the end of first line, goes to the
+   second one and then write some text to replace that. The span tag carries
+   along with it a style=font-size:1.6em;, due to which the text is enlarged.
+   This size is taken similar to font-size in the body element. There is as
```

Comment:
This size seems to be taken from the font-size from the body element's CSS class, but we do not know what JS code is causing it to be inlined in the span tag.

Reply:
Done!